CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 ## Solidus 3.0.0 (master, unreleased)
 
+Migrated default billing addresses to address book
```

Comment:
This should not go into the 3.0 changelog, we need to backport this task into 2.11.x. I'll take care of the changelog, you can remove this entirely.

Reply:
Okay, I will remove this when I add the test case.